pytest_catchlog/fixture.py

```diff
@@ -16,6 +16,9 @@ class LogCaptureFixture(object):
     def __init__(self, item):
         """Creates a new funcarg."""
         self._item = item
+        # set logging levels to facilitate use without needing to import logging
+        for level in ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']:
+            setattr(self, level, getattr(logging, level))
```

Comment:
Good to consider custom log levels. I did some digging (and by that learning). The ``logging`` module indeed allows registering log levels via ``logging.addLevelName(lvl, name``, but... levels are registered in a 'protected' dict and not exposed as attributes. A level is just an integer and the purpose of registering of levels by label is only to have that label appear in log records.

There is a ``logging.getLevelName(lvl)`` method but not a ``getAllLevels()``.

Commands below summarize how custom levels work. Note that in python3 there's no ``_levelNames`` but instead there are ``_levelToName`` and ``_nameToLevel``.

```
>>> import sys
>>> sys.version_info
sys.version_info(major=2, minor=7, micro=12, releaselevel='final', serial=0)
>>> import logging
>>> logging.basicConfig(format='%(levelname)s:%(message)s')
>>> logging.addLevelName(45, 'custom')
>>> logging.getLevelName(45)
'custom'
>>> logging.log(30, 'message')
WARNING:message
>>> logging.log(45, 'message')
custom:message
>>> logging.log(46, 'message')
Level 46:message
>>> logging.WARNING
30
>>> logging.custom
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'custom'
>>> logging._levelNames
{0: 'NOTSET', 10: 'DEBUG', 'WARN': 30, 20: 'INFO', 'ERROR': 40, 'DEBUG': 10, 30: 'WARNING', 'INFO': 20, 'WARNING': 30, 40: 'ERROR', 45: 'custom', 'custom': 45, 50: 'CRITICAL', 'CRITICAL': 50, 'NOTSET': 0}
```

Based on this I'd say facilitating custom levels is far-fetched. As levels are merely numbers I'll add a test for a custom numeric level.

Leaves the actual implementation. On a first glance it indeed doesn't look like very good practice to list these levels, on the other hand, it _is_ very explicit and the levels aren't really subject to change. 

@abusalimov Let me know if you still want the implementation changed. My opinion is that changing it doesn't improve, but it's not my project. ;)